.env.sample

```diff
@@ -1,4 +1,6 @@
-DISCORD_CLIENT_ID=1082239786162606080
-NEXT_PUBLIC_DISCORD_CLIENT_ID=1082239786162606080
-DISCORD_CLIENT_SECRET=123
-DB_PASSWORD=password
+# Secrets
+AUTH_SECRET=secret
+
+DISCORD_CLIENT_SECRET=
```

Comment:
Should the DISCORD_CLIENT_SECRET be filled out here since it was filled out before? AUTH_SECRET is still filled out

Reply:
It's going to fail the same being empty as it does as `123` and I believe leaving it blank is more indicative that it needs to be filled in for certain features to work.

`AUTH_SECRET` is used when generating session tokens. It could probably be left empty too as I believe `envalid` would convert it to an empty string. However, it's mainly only important on prod so setting a default for development isn't a concern.

Reply:
Updated all default secrets to be empty, populated the sample env with common development values.

Reply:
That makes sense now, I didn't know what a client secret was before. thought it worked the same way AUTH_SECRET did.